pkg/metrics/metrics_test.go

```diff
@@ -1,20 +1,25 @@
 package metrics
 
 import (
+	"bytes"
+	"github.com/prometheus/client_golang/prometheus"
 	promtestutil "github.com/prometheus/client_golang/prometheus/testutil"
 	clientmodel "github.com/prometheus/client_model/go"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"net/http"
 	"strings"
 	"testing"
 	"time"
 )
 
 func TestHistogramVec(t *testing.T) {
 	// given
+	reg := prometheus.NewRegistry()
 	m := newHistogramVec("test_histogram_vec", "test histogram description", "status_code", "kube_verb")
 	getSuccess, getFailure, listSuccess, listFailure := getExpectedLabelPairs()
-	RegisterCustomMetrics()
+	//RegisterCustomMetrics()
```

Comment:
```suggestion
```

Reply:
I always forget to use the "commit suggestion" feature 😅 
[94a372a](https://github.com/codeready-toolchain/registration-service/pull/352/commits/94a372ad764a62a01e8276e3456d31c9941b88d4) done